api.ts

```diff
@@ -1220,12 +1220,13 @@ export function makeMetadataFormula(
     // Formula context is serialized here because we do not want to pass objects into
     // regular pack functions (which this is)
     execute([search, serializedFormulaContext], context) {
-      let formulaContext = {};
+      let formulaContext: any = {};
       try {
         formulaContext = JSON.parse(serializedFormulaContext);
       } catch (err: any) {
         //  Ignore.
       }
+      formulaContext.__brand = 'MetadataContext';
```

Comment:
So the id with brands is that they should only exist within the type system and not affect anything at runtime. Usually you'd do this with a cast.

Here I think you can just declare `let formulaContext = {} as MetadataContext;`

Reply:
Done, and testing in coda repo now...